### ISSUE TITLE
McAfee blocking my game from launching, won't stop putting it in quarantine. (Bleeding Edge)

### ISSUE THREAD

**Issue by Kawaiitron** [BepInEx/BepInEx#781]
### What happened?

I installed Bleeding Edge from this link: https://builds.bepinex.dev/projects/bepinex_be
I wanted to play Digimon World: Next Order with some mods, so I followed a tutorial on how to install it and it said it requires BepInEx Bleeding Edge. I promptly installed the one labeled "BepInEx Unity (IL2CPP) for Windows (x64) games" under artifact #688.
I got everything set up, but whenever I launch my game, McAfee shoves the modded UnityEngine.CoreModule.dll into quarantine for being a "threat."
I'm no good at analyzing source code, and I do tend to be a little paranoid with technology, so I was hesitant to unquarantine the file at first, but after looking online for a bit, I decided I was definitely just being paranoid and it was indeed a false positive. So I unquarantine the file because I want to play Next Order again, but guess what? McAfee just shoves it right back into quarantine. Every time. I don't know how to make it stop, nor how to uninstall Bleeding edge if push comes to shove.
I guess this is more of a McAfee troubleshoot, but my pea brain thought it would be a good idea to ask here. Apologies if I can't be helped, I'm hopeless at tech sometimes. :'D
Also, apologies if I don't respond right away, I sometimes forget about posts like this on top of being a college student (although it's the weekend as of the time I'm writing this, so class won't be an issue).

### BepInEx Distribution

Bleeding Edge from BepisBuilds

### Log outputs

```txt
It says all this before my PC has the "[file].exe has stopped working" popup that Windows gives (as McAfee proceeds to quarantine UnityEngine.CoreModule.dll:
[Message: Preloader] BepInEx 6.0.0-be.688 - Digimon World Next Order
[Message: Preloader] Built from commit 49015217f3becf052d33fa4658ac19229f5daa3a
[Info   :   BepInEx] System platform: Windows 10 64-bit
[Info   :   BepInEx] Process bitness: 64-bit (x64)
[Info   :   BepInEx] Running under Unity 2019.4.11f1
[Info   :   BepInEx] Runtime version: 6.0.7
[Info   :   BepInEx] Runtime information: .NET 6.0.7
[Info   : Preloader] 0 patcher plugins loaded
[Info   : Preloader] 0 assemblies discovered
[Message:AssemblyPatcher] Executing 0 patch(es)
[Message:   BepInEx] Chainloader initialized

Also, if it helps, McAfee lists the threat as "ti!6D083E8FD949". No idea what that means, peanut brain that knows no coding. :'D
```

### Environment

```markdown
- OS: Windows 10
- BepInEx: I don't understand how to open dll files to check, but it's the latest version of Bleeding Edge as of 2/17/24.
- Game: Digimon World: Next Order
```

**Comment by jinko690** [comment]
Your problem is Mcafee. You shouldn't be using it. Windows defender is enough.

**Comment by Mukul1127** [comment]
> Your problem is Mcafee. You shouldn't be using it. Windows defender is enough.

Agreed, Windows Defender is more than enough and might even block more that McAfee
